src/components/Display.js

```diff
@@ -0,0 +1,11 @@
+import { PropTypes } from 'prop-types';
+
+function Display({ result = '0' }) {
```

Comment:
## Enforce the use of `ES6` syntax
- [ ] Kindly change this function declaration into an arrow function.